src/www/ui/api/documentation/openapi.yaml

```diff
@@ -14,7 +14,7 @@ openapi: 3.0.2
 info:
   title: FOSSology API
   description: Automate your fossology instance using REST API
-  version: 1.0.13
+  version: 1.0.14
```

Comment:
Hello @deveaud-m,
Due to merge of another PR, the version of API has changed to `1.0.14` (https://github.com/fossology/fossology/blob/master/src/www/ui/api/documentation/openapi.yaml#L17)

I can't understand why it didn't cause a merge conflict, but would it be okay it you change the version to `1.0.15`?